src/ToDoList/ToDoList.tsx

```diff
@@ -34,6 +51,9 @@ export const ToDoList = () => {
                 <div className='header-and-column'>
                     <h1 className='column-header'>TO DO</h1>
                     <div className='column'>
+                        {toDoTasks.map((it) => {
+                            return <TaskCard task={it} />
```

Comment:
1. You should provide `key` prop to each item when render components via `.map()`.
2. Unnecessary return in arrow function.